src/bin/dns/main.rs

```diff
@@ -41,6 +44,12 @@ fn main() {
         .long("port")
         .takes_value(true),
     )
+    .arg(
+      clap::Arg::with_name("secrets-file")
+        .short("sf")
+        .long("secrets-file")
+        .takes_value(true)
+    )
```

Comment:
FYI we're still figuring out the cli and distributed-fly production setup so I expect this to change, but it'll be fine for now.